cellpack/autopack/Compartment.py

```diff
@@ -187,15 +182,14 @@ def __init__(self, name, object_info):
         # if a highres vertices is provided this give the surface point,
         # not the one provides
         # to compute inside points.
-        self.isBox = object_info["bounding_box"] is not None
-        self.bounding_box = (
-            object_info["bounding_box"]
-            if object_info["bounding_box"] is not None
-            else None
+        self.is_sphere = object_info["type"] == "single_sphere"
+        self.isBox = (
+            "bounding_box" in object_info and object_info["bounding_box"] is not None
         )
-        self.isOrthogonalBoundingBox = (
-            1 if object_info["bounding_box"] is not None else None
+        self.bounding_box = (
+            object_info["bounding_box"] if "bounding_box" in object_info else None
         )
+        self.isOrthogonalBoundingBox = 1 if "bounding_box" in object_info else None
```

Comment:
should we also change this to snake case for consistency?
```suggestion
        self.is_orthogonal_bounding_box = 1 if "bounding_box" in object_info else None
```